<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticeWidget</class>
 <widget class="QWidget" name="PracticeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_N">
    <item>
     <widget class="QLabel" name="label_N">
      <property name="text">
       <string>N=</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinbox_N">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>591</width>
     <height>257</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="table_matrixItems">
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>l_i_1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>l_i_2</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layout_solutionAndRedirect">
      <item>
       <layout class="QHBoxLayout" name="layout_nonRandomizedSolution">
        <item>
         <widget class="QRadioButton" name="radioButton_nonRandomizedSolution">
          <property name="text">
           <string>нерандомізований розв'язок</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_nonRandomizedSolution_index">
          <property name="text">
           <string>i:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_nonRandomizedSolution_index">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_randomizedSolution">
        <item>
         <widget class="QRadioButton" name="radioButton_randomizedSolution">
          <property name="text">
           <string>рандомізований розв'язок</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_randomizedSolution_results_i_index">
            <property name="text">
             <string>i:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox_randomizedSolution_results_i_index">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_randomizedSolution_results_i_probability">
            <property name="text">
             <string>вірогідність i-тої:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_randomizedSolution_results_i_probability">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_randomizedSolution_results_j_index">
            <property name="text">
             <string>j:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_randomizedSolution_results_j_index">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_randomizedSolution_results_j_probability">
            <property name="text">
             <string>вірогідність j-тої:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_randomizedSolution_results_j_probability">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_buttons">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="layout_correct">
            <item>
             <widget class="QLabel" name="label_correctName">
              <property name="text">
               <string>Правильних відповідей:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_correctValue">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="button_generate">
            <property name="text">
             <string>Інше завдання</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="layout_incorrect">
            <item>
             <widget class="QLabel" name="label_incorrectName">
              <property name="text">
               <string>Неправильних відповідей:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_incorrectValue">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="button_verify">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Перевірити</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="button_toSolverWithCurrentTask">
        <property name="text">
         <string>Показати розв'язок задачі</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_toTheory">
        <property name="text">
         <string>До теорії</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Модуль практики</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_error</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
